src/test/resources/dbpedia_test.owl

```diff
@@ -0,0 +1,2919 @@
+<rdf:RDF xml:base="http://dbpedia.org/ontology/" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:cc="http://creativecommons.org/ns#" xmlns:vann="http://purl.org/vocab/vann/" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:wgs84pos="http://www.w3.org/2003/01/geo/wgs84_pos#" xmlns:cidoccrm="http://purl.org/NET/cidoc-crm/core#" xmlns:wikidata="http://www.wikidata.org/entity/" xmlns:dul="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#" xmlns:d0="http://www.ontologydesignpatterns.org/ont/d0.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:prov="http://www.w3.org/ns/prov#" xmlns="http://dbpedia.org/ontology/">

```

Comment:
Could you please reduce the size of this file and focus on the statements which are really necessary for the test?